components/tikv_util/build.rs

```diff
@@ -0,0 +1,170 @@
+// Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
+
+// Reference: https://github.com/larksuite/perf-monitor-rs/blob/master/build.rs
```

Comment:
Is it possible to use `perf-monitor-rs` directly?

Reply:
I'd like to manipulate the raw API but `perf-monitor-rs` provides a high abstraction. If using `perf-monitor-rs`, I have to do the reverse of many of the operations it does.